astro/src/content/docs/operate/deploy/proxy-setup.mdx

```diff
@@ -144,6 +145,10 @@ This guide covers proxying requests *to* FusionAuth, in order to add a layer of
 
 If you are self-hosting, you can also proxy requests *from* FusionAuth, such as [webhooks](/docs/extend/events-and-webhooks/) or [connector requests](/docs/lifecycle/migrate-users/connectors/). You can read more about that in the [Configuration Reference](/docs/reference/configuration). Look for the `proxy.*` configuration values. This functionality is not available in FusionAuth Cloud.
 
+## SNI and FusionAuth Cloud Instances
+
```

Comment:
Please add this to the cloud limits too. 

https://github.com/FusionAuth/fusionauth-site/blob/main/astro/src/content/docs/get-started/run-in-the-cloud/_cloud-limits.mdx

You could add a single line to the include file and link over to this section.